docs/blocks-dynamic.md

```diff
@@ -0,0 +1,77 @@
+It is also possible to create dynamic blocks. These are blocks that can change their content even if the post is not saved. One example from WordPress itself is the latest posts block. This block will update everywhere it is used whena new post is published.
```

Comment:
Should we add a "title" for the page.

"It is also possible ..." suggest that we read something before. Is this the case? or should we drop "also"

"whena" should be "when a"